src/database/models/version_item.rs

```diff
@@ -3,19 +3,37 @@ use crate::database::Result;
 use bson::{Bson, Document};
 use serde::{Deserialize, Serialize};
 
+//TODO: Files should probably be moved to their own table
 #[derive(Deserialize, Serialize)]
 pub struct Version {
     pub version_id: i32,
     pub mod_id: i32,
-    pub title: String,
-    pub changelog_path: String,
-    pub files_path: Vec<String>,
+    pub name: String,
+    pub number: String,
+    pub changelog_url: Option<String>,
     pub date_published: String,
-    pub author: String,
     pub downloads: i32,
-    pub dependencies: Vec<String>,
+    pub files: Vec<VersionFile>,
+    pub dependencies: Vec<i32>,
```

Comment:
What specifically does this ID reference?  It may be useful to add documentation / doc comments, since we don't have the type wrappers.

Reply:
The Version ID of the dependencies. I'll add docs